### ISSUE TITLE
Project import from ssh bitbucket creates invalid name

### ISSUE THREAD

**Issue by jgroom33** [eclipse-che/che#8688]
### Reproduction Steps
Workspace - import project from ssh Git

ERROR 1: Project configuration window hangs

![image](https://user-images.githubusercontent.com/3970574/36012816-f74fd2cc-0d1d-11e8-9294-adecafb0d401.png)

ERROR 2: The name value is invalid.  It cannot contain `/`

This character causes the 'Workspace -> Create factory ...' action to fail

Resulting relevant config:

```json
    {
      "links": [],
      "name": "/bp-ra-akamai-aura",
      "attributes": {},
      "type": "blank",
      "source": {
        "location": "ssh://git@bitbucket.ciena.com/bp_ra/bp-ra-akamai-aura.git",
        "type": "git",
        "parameters": {}
      },
      "path": "/bp-ra-akamai-aura",
      "problems": [],
      "mixins": []
    }
```

WORKAROUND:
Copy config from workspace into a new factory and remove the leading `/`

**OS and version:**
Ubuntu 16.04
Che 6.0.0

**Diagnostics:**

**Comment by eivantsov** [comment]
@jgroom33 I have hard time reproducing it. Is this still an issue if you use the latest nightly tag?

**Comment by jgroom33** [comment]
this appears to be resolved in 6.2
